src/lib/colourType.js

```diff
@@ -12,4 +13,4 @@ export const isHex = colour => {
 
 export const isRGBorHSL = colour => /^(rgb|hsl)a?\(.*?\)/.test(colour);
 
-export const isKeyword = colour => has(colourNames, colour.toLowerCase());
+export const isKeyword = colour => has.call(colourNames, colour.toLowerCase());
```

Comment:
The point is that it is not needed, even [IE6 supports `.call`](https://msdn.microsoft.com/en-us/library/h2ak8h2y(v=vs.94).aspx), and as said in the other thread, all versions of node does support all three function methods.